prov/verbs/src/verbs_mr.c

```diff
@@ -94,9 +94,9 @@ fi_ibv_mr_reg_ibv_mr(struct fi_ibv_domain *domain, void *buf,
 }
 
 static inline
-int fi_ibv_mr_dereg_ibv_mr(struct ibv_mr *mr)
+int fi_ibv_mr_dereg_wrapper(struct ibv_mr *mr)
 {
-	return -ibv_dereg_mr(mr);
+	return mr ? -ibv_dereg_mr(mr) : 0;
```

Comment:
I would remove the wrappers, along with the VERBS_EXP support.

Reply:
Having the wrapper avoids duplicating the non-NULL check for `mr`.

Reply:
It's still a 1-line function only called from 2 places.  I would still remove this, but it isn't a big deal.  We should see whether the cache handles 0 length regions or if it makes sense to store those there.